### PR TITLE
Fix the incorrect ammo count in the China Lake bundle's description

### PR DESCRIPTION
**Pull request by liltenhead**
## About the PR

Like the Bulldog bundle, it didn't count up the ammo present in the gun itself as a part of the total. This brings it up to standard with the rest of the bundles. 

**Media**

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- fix: Fixed the incorrect ammo count in the China Lake bundle's description.

### CODE REVIEW

**Comment by ElectroJr** [comment]
Test fail is erroneous. Should fix itself if you merge master.

File: Resources/Locale/en-US/store/uplink-catalog.ftl [hunk 1]
```diff
@@ -153,7 +153,7 @@ uplink-buldog-bundle-name = Bulldog Bundle
 uplink-buldog-bundle-desc = Lean and mean: Contains the popular Bulldog Shotgun, a 12g beanbag drum and three 12g buckshot drums.
 
 uplink-grenade-launcher-bundle-name = China-Lake Bundle
-uplink-grenade-launcher-bundle-desc = An old China-Lake grenade launcher bundled with 8 rounds of various destruction capability.
+uplink-grenade-launcher-bundle-desc = An old China-Lake grenade launcher bundled with 11 rounds of various destruction capability.
```
Comment (metalgearsloth) [inline Resources/Locale/en-US/store/uplink-catalog.ftl:156]: Honestly I think putting hardcoded values into descriptions is silly and we should just remove the number entirely.

Comment (liltenhead) [reply Resources/Locale/en-US/store/uplink-catalog.ftl:156]: I'd do that if I could but I wouldn't have any idea how to do that unfortunately. I can stop fixing values if there is a plan to fix it permanently.

Comment (Partmedia) [inline Resources/Locale/en-US/store/uplink-catalog.ftl:156]: Would something like this work?
```suggestion
uplink-grenade-launcher-bundle-desc = An old China-Lake grenade launcher. Shells included!
```

Comment (liltenhead) [reply Resources/Locale/en-US/store/uplink-catalog.ftl:156]: Yeah, could make the descriptions generic, but it's better players know exactly what they are buying in my opinion. It seems a more permanent solution is wanted than individually fixing descriptions of what contents are inside a bundle.

**Pull request merged**